logging/google/cloud/logging/handlers/app_engine.py

```diff
@@ -67,3 +71,17 @@ def get_gae_resource(self):
             },
         )
         return gae_resource
+
+    def get_gae_labels(self):
+        """Return the labels for GAE app which includes trace_id.
```

Comment:
The name for `trace_id` in labels should be `appengine.googleapis.com/trace_id`, which is required in the logging documents (I forget which doc mentions this but will find it later). And the UI is doing correlation for logs with same trace_id based on this label name.